config/templates/cluster.yaml

```diff
@@ -165,8 +165,12 @@ kmsKeyArn: "{{.KMSKeyARN}}"
 # experimental:
 #   nodeDrainer:
 #     enabled: true
+#   awsEnvironment:
+#     enabled: true
+#     environment:
+#       CFNSTACK: '{ "Ref" : "AWS::StackId" }'
```

Comment:
Could be. I personally like this kind of freedom/power in exchange for clarity but definitely need more thinking. Anyways, something like `CFNSTACK: 'AWS::StackId' won't be fundamentally different from your current implementation, right?

Do we need the new `advanced:` section in cluster.yaml when graduation from experimental? 😈